src/lib.rs

```diff
@@ -2197,15 +2316,15 @@ impl<'tu> Entity<'tu> {
         ) -> CXChildVisitResult {
             unsafe {
                 let &mut (tu, ref mut callback) =
-                    &mut *(data as *mut (&TranslationUnit, Box<EntityCallback>));
+                    &mut *(data as *mut (&TranslationUnit, Box<dyn EntityCallback>));
```

Comment:
The `dyn` trait work was added so that the package compiles cleanly on current Rust stable (1.37)